scripts/auto-rebase/rebase.sh

```diff
@@ -688,7 +688,7 @@ rebase_to() {
         exit 1
     fi
 
-    rebase_branch="rebase-${ver_stream}+amd64-${amd64_date}+arm64-${arm64_date}"
+    rebase_branch="rebase-${ver_stream}_amd64-${amd64_date}_arm64-${arm64_date}"
```

Comment:
I noticed that `+` are not working good with prow's github query as they're used to join filters together, [example](https://prow.ci.openshift.org/pr?query=is%3Apr+repo%3Aopenshift%2Fmicroshift+author%3Amicroshift-rebase-script+head%3Arebase-4.12.0-0.nightly%2Bamd64-2023-01-10-062211%2Barm64-2023-01-12-221036):
```
is:pr+repo:openshift/microshift+author:microshift-rebase-script+head:rebase-4.12.0-0.nightly+amd64-2023-01-10-062211+arm64-2023-01-12-221036
```